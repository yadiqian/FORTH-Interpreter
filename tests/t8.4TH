t . CR 4 . CR 5 6 / . CR 345.0 2.432432 10 ^ . CR .