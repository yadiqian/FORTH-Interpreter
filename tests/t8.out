t
4
0
7251.1514
345.0